### PR TITLE
fix(ONYX-1013): update images and styling

### PR DESCRIPTION
**Pull request by tam-kis**
The type of this PR is: Fix

This PR solves [ONYX-1013](https://artsyproduct.atlassian.net/browse/ONYX-1013)

### Description
Update and crop images, refactor spacing and update for mWeb.

Screenshot 2024-06-25 at 13 37 46

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-1013]: https://artsyproduct.atlassian.net/browse/ONYX-1013?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Sell/Components/SubmissionLayout.tsx [hunk 1]
```diff
@@ -31,13 +31,13 @@ export const SubmissionLayout: React.FC<SubmissionLayoutProps> = ({
...
             <Column span={[8]}>
-              <Box maxWidth="100vw" width={CONTENT_WIDTH} p={2} pt={4}>
+              <Box maxWidth={CONTENT_WIDTH} p={2} pt={[2, 4]}>
                 {children}
               </Box>
             </Column>
           </GridColumns>
         ) : (
-          <Box maxWidth="100vw" width={CONTENT_WIDTH} p={2} pt={4}>
+          <Box maxWidth={CONTENT_WIDTH} p={2} pt={[2, 4]}>
```
Comment (tam-kis) [inline src/Apps/Sell/Components/SubmissionLayout.tsx:40]: `maxWidth={CONTENT_WIDTH}` is enough for responsiveness.

**Review by olerichter00 — APPROVED** [review]
Thanks 🙏

**Comment by dzucconi** [comment]
Not needed but It might be simpler for your layout to use a full fluid responsive image: https://www.notion.so/artsy/Handling-Images-On-The-Web-In-Force-5c6832bf4b3e431cb4830061ce057399?pvs=4#be9ce98052a0414d80ae5cb1b738bc06

File: src/Apps/Sell/Components/SubmissionLayout.tsx (hunk 1 above)
Comment (olerichter00) [reply src/Apps/Sell/Components/SubmissionLayout.tsx:40]: This change is causing layout issues on the title step. I opened https://github.com/artsy/force/pull/14081 to revert it. Was there a specific reason for this change?

 
Screenshot 2024-06-25 at 21 28 12

**Pull request merged**